arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/OpCode.java

```diff
@@ -211,4 +213,14 @@ public boolean isCall() {
         || this == OpCode.DELEGATECALL
         || this == OpCode.STATICCALL;
   }
+
+  /**
+   * Matches if the current {@link OpCode} is contained within a list of {@link OpCode}s.
+   *
+   * @param opCodes list of {@link OpCode}s to match against.
+   * @return if the current {@link OpCode} is contained within the list.
+   */
+  public boolean isAnyOf(OpCode... opCodes) {
+    return Arrays.asList(opCodes).contains(this);
```

Comment:
I don't like this, it is an innocuous looking function that will actually allocate a new list at every call.

Let us just go with a simple for loop on the variadic arguments.

Reply:
OK, I will change it.